src/flp.rs

```diff
@@ -104,6 +106,11 @@ pub enum FlpError {
     #[error("Field error: {0}")]
     Field(#[from] FieldError),
 
+    #[cfg(feature = "experimental")]
+    /// An error happened during noising.
+    #[error("differential privacy error: {0}")]
+    DifferentialPrivacy(#[from] crate::dp::DpError),
```

Comment:
Suggestion: To match the name of the module? Just an idea, maybe not a good one :)
```suggestion
    Dp(#[from] crate::dp::DpError),
```